lib/new_relic/agent/llm/chat_completion.rb

```diff
@@ -0,0 +1,15 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic
+  module Agent
+    module Llm
+      module ChatCompletion
```

Comment:
After taking another look at the attributes list, it seems that fewer unique attributes are shared among ChatCompletionSummary and ChatCompletionMessage than we originally thought. Since there's only one, I thought this would be better suited as a module, and then we can simplify the inheritance for the classes that actually create events.

Reply:
Ah! I didn't realize so few were shared. I like this move.